helm/app-operator/templates/service.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ include "resource.default.namespace" . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+    giantswarm.io/monitoring: "true"
```

Comment:
I think this label is for tenant cluster, doesn't it?

Reply:
We decided to use the same labels for control-plane components

Reply:
That way, we can use only one service monitor